packages/website/src/components/Curriculum/TechSliding.tsx

```diff
@@ -15,8 +15,15 @@ type TechSlidingProps = {
   isAnimationReverse?: 'reverse' | 'normal';
 };
 
+const sliding = keyframes`
+   100% {
+    transform: translateX(-66.66%);
+  }
+`;
+const animation = `${sliding} linear infinite`;
+
 export const TechSliding = ({ techIconsArray = techNamesTop, isAnimationReverse = 'normal' }: TechSlidingProps) => {
```

Comment:
💭 I missed it in the last review. It would be great if we remove `techIconsArray = techNamesTop` and require `techIconsArray` to be provided. This way it will be better visible from the parent level where the data comes from.